docs/docs/images/forum-round.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="46px" height="46px" viewBox="0 0 46 46" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(73.333333%,73.333333%,73.333333%);fill-opacity:1;" d="M 38.8125 29.621094 L 43.664062 29.621094 L 43.664062 45.898438 L 0 45.898438 L 0 29.621094 L 4.851562 29.621094 L 4.851562 41.828125 L 38.8125 41.828125 Z M 38.8125 29.621094 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(96.078431%,50.196078%,14.509804%);fill-opacity:1;" d="M 10.144531 28.402344 L 33.960938 32.617188 L 34.972656 28.621094 L 11.15625 24.40625 Z M 13.273438 18.855469 L 35.328125 27.476562 L 37.398438 23.777344 L 15.347656 15.15625 Z M 19.40625 9.757812 L 38.105469 22.816406 L 41.238281 19.671875 L 22.535156 6.613281 Z M 31.488281 0.101562 L 27.566406 2.542969 L 42.074219 18.929688 L 46 16.488281 Z M 9.703125 37.757812 L 33.960938 37.757812 L 33.960938 33.691406 L 9.703125 33.691406 Z M 9.703125 37.757812 "/>
+</g>
+</svg>
```

Comment:
By optimizing this SVG this can be reduced to:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="46" height="46">
	<path d="M38.813 29.621h4.851v16.277H0V29.621h4.852v12.207h33.96Zm0 0" style="stroke:none;fill-rule:evenodd;fill:#bbb;fill-opacity:1"/>
	<path d="m10.145 28.402 23.816 4.215 1.012-3.996-23.817-4.215Zm3.128-9.547 22.055 8.622 2.07-3.7-22.05-8.62Zm6.133-9.097 18.7 13.058 3.132-3.144L22.535 6.613ZM31.488.102l-3.922 2.441L42.074 18.93 46 16.488ZM9.703 37.758h24.258V33.69H9.703Zm0 0" style="stroke:none;fill-rule:nonzero;fill:#f58025;fill-opacity:1"/>
</svg>
```